charts/flink-job/templates/operator-role.yaml

```diff
@@ -0,0 +1,13 @@
+# This role is granted to oncall engineers to allow modification of configmaps,
+# which we do not generally want to permit, but in namespaces that are running
+# flink, oncall engineers sometimes need to manually edit the configmaps that
+# the flink operator uses to persist state about the flink cluster.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-operator
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["configmaps"]
+  verbs: ["patch"]
```

Comment:
do we have delete permission?

Reply:
do you need to delete these configmaps or just edit the values in them?

before we start just deleting these I think we should understand what the data in them is and how the flink operator is using it.....

Reply:
The one we want (need) to delete is the orphaned configmap that generated quickly and dynamically when the app was bouncing. When we force stop / force sync (to either simulate "redeploy" operation or simply want a clean state)  the app, sometimes an orphaned configmap will be left there. We have seen this might cause weird behavior in the next run of the job. So if we can delete them, it would give us a clean state to start with.

Reply:
![Screen Shot 2021-09-21 at 5 33 49 PM](https://user-images.githubusercontent.com/12125887/134426193-d627eb2e-6fb5-4e56-ad7b-30c297e222ab.png)
one of the `interaction-store-<id>-ledger` should be removed. And the dispatcher ledger can be patched or removed.